src/domain/session/RoomGridViewModel.js

```diff
@@ -183,6 +189,7 @@ export class RoomGridViewModel extends ViewModel {
 
 import {createNavigation} from "../navigation/index.js";
 import {ObservableValue} from "../../observable/ObservableValue.js";
+import { Segment } from "../navigation/Navigation.js";
```

Comment:
Nit: only the imports needed for the tests should be below here, the others up at the top. You can also avoid the import by doing `this.navigation.segment(...)` fwiw.